### ISSUE TITLE
How about replacing the CC loss with triplet loss?

### ISSUE THREAD

**Issue by mouxingyang** [DoubtedSteam/MPANet#3]
Hi~
It's a nice job. The proposed CC loss seems to narrow the modality gap, which is often done by triplet loss. So, how about replacing CC with the triplet loss?

Thanks~

**Comment by swb-upc** [comment]
I compared the two loss functions. CC loss is better  than triplet loss In terms of performance.

**Comment by swb-upc** [comment]
Can you give a pre-trained model for comparison? My training did not achieve the results in the paper.

**Comment by mouxingyang** [comment]
> > Hi~
> > It's a nice job. The proposed CC loss seems to narrow the modality gap, which is often done by triplet loss. So, how about replacing CC with the triplet loss?
> > Thanks~
> 
> I compared the two loss functions. CC loss is better than triplet loss In terms of performance.

Did you record the ablation results between them? Could you please give me the results? Thanks~

**Comment by mouxingyang** [comment]
In my implementation, MPANet with cc loss exceeds that with triplet loss by nearly 10% in terms of the performance. I really wonder why the cc loss could help the performance improvement. Thanks~~

**Comment by swb-upc** [comment]
加个QQ讨论一下吧。1092654533


| |
邵文斌
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/27/2021 ***@***.***> wrote：

In my implementation, MPANet with cc loss exceeds that with triplet loss by nearly 10% in terms of the performance. I really wonder why the cc loss could help the performance improvement. Thanks~~
